nimare/io.py

```diff
@@ -275,3 +279,87 @@ def convert_sleuth_to_dataset(text_file, target="ale_2mm"):
         )
     dict_ = convert_sleuth_to_dict(text_file)
     return Dataset(dict_, target=target)
+
+
+def convert_neurovault_to_dataset(collection_ids, contrast, img_dir=None):
+    """
+    Convert a group neurovault collections into a NiMARE Dataset.
+
+    Parameters
+    ----------
+    collection_ids : :obj:`list` of :obj:`int`
+        A list of collections on neurovault specified by their id.
+        The collection ids can accessed through the neurovault API
+        (i.e., https://neurovault.org/api/collections) or
+        their main website (i.e., https://neurovault.org/collections).
+        For example, in this URL https://neurovault.org/collections/8836/,
+        `8836` is the collection id.
+    contrast : :obj:`str`
+        String representing the name of the contrast you wish add to the dataset.
+        For example, under the `Name` column in this URL https://neurovault.org/collections/8836/,
+        a valid contrast could be "as-Animal"
+    img_dir : :obj:`str` or None
+        Base path to save all the downloaded images, by default the images
+        will be saved to a temporary directory with the prefix "neurovault"
+
+    """
+    if img_dir is None:
+        img_dir = Path(tempfile.mkdtemp(prefix="neurovault_"))
+    else:
+        img_dir = Path(img_dir)
+        img_dir.mkdir(parents=True, exist_ok=False)
+
+    map_type_conversion = {
+        "T map": "t",
+        "variance": "varcope",
```

Comment:
I'm not sure if we can assume that NeuroVault "variance" maps correspond to varcopes.